drivers/hmis/lib/tasks/graphql.rake

```diff
@@ -18,6 +18,40 @@ task dump_graphql_schema: [:environment, 'log:info_to_stdout'] do
   end
 end
 
+desc 'Generate GraphQL Enums'
+task generate_graphql_enums: [:environment, 'log:info_to_stdout'] do
+  source = File.read('drivers/hmis/lib/hud/hud_lists.json')
+  skipped = ['race', 'gender', '2.4.2', '1.6']
+  output_dir = 'drivers/hmis/app/graphql/types/hmis_schema/enums/hud'
+
+  JSON.parse(source).each do |element|
+    next if skipped.include?(element['code'].to_s)
+
+    name = element['name']
+    arr = []
+    arr.push "###\n# Copyright 2016 - 2022 Green River Data Analysis, LLC\n#\n# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md\n###\n\n# frozen_string_literal: true\n"
```

Comment:
We should pull this from `lib/tasks/code.rake`  Might need to move that copyright definition somewhere we can access across rake tasks.

Reply:
https://github.com/greenriver/hmis-warehouse/pull/2601/commits/dc5c7987d4a9b1c3d5fe4cace5e6dccbc6090a56